dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,376 @@
+                        }
+                    ]
+                },
+                {
+                    "title": "Claim",
+                    "text": [
+                        "In Dataland, ",
+                        "you",
+                        " have the power."
+                    ]
+                },
+                {
+                    "title": "Campaigns",
+                    "text": [
+                        "Create or join a campaign",
+                        " to get the missing data."
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Crop-growth.svg",
+                            "title": "SFDR",
+                            "text": "101",
+                            "date": "14th of August"
+                        },
+                        {
+                            "icon": "/static/icons/Diagram--reference.svg",
+                            "title": "EU Taxonomy",
+                            "text": "269"
+                        },
+                        {
+                            "icon": "/static/icons/Dashboard--reference.svg",
+                            "title": "LksG",
+                            "text": "101",
+                            "date": "14th of August"
+                        },
+                        {
+                            "icon": "/static/icons/Explore.svg",
+                            "title": "Pathway to Paris",
+                            "text": "101",
+                            "date": "Starting soon!"
+                        }
+                    ]
+                },
+                {
+                    "title": "Come together",
+                    "text": [
+                        "Collaborate on datasets ",
+                        "to share the costs."
+                    ],
+                    "image": [
+                        "Join now!"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Face--add.svg",
+                            "title": "10",
+                            "text": "Companies joining every day"
+                        },
+                        {
+                            "icon": "/static/icons/3d-mpr-toggle.svg",
+                            "title": "9000",
+                            "text": "European Companies"
+                        },
+                        {
+                            "icon": "/static/icons/Certificate.svg",
+                            "title": "3",
+                            "text": "Active Campaigns"
+                        },
+                        {
+                            "icon": "/static/icons/Area.svg",
+                            "title": "5",
+                            "text": "Frameworks"
+                        }
+                    ]
```

Comment:
These KPIs should not all be hardcoded right? Should we postpone displaying them until we have to possibility to dynamically derive them?

Reply:
They are in a component that won't be included in the current landing page. In the future version, we'll probably read the data from an API

Reply:
Yes sounds reasonable 👍 .

Then I think we should remove it from the `content.json` so that we have only stuff in there that is actually mounted somewhere.